### PR TITLE
[5.2] Dry up managers resolvers

### PR DESCRIPTION
**Pull request by lucasmichot**
Dry up managers resolvers.

---

Nothing functional was changed.

### CODE REVIEW

File: src/Illuminate/Auth/AuthManager.php [hunk 1]
```diff
@@ -80,23 +80,19 @@ public function guard($name = null)
      */
     protected function resolve($name)
     {
-        $config = $this->getConfig($name);
-
-        if (is_null($config)) {
+        if (is_null($config = $this->getConfig($name))) {
```
Comment (GrahamCampbell) [inline src/Illuminate/Auth/AuthManager.php]: I think we were intentionally avoiding inline assignment here?

**Comment by GrahamCampbell** [comment]
Just got one review comment. :)

File: src/Illuminate/Auth/AuthManager.php (hunk 1 above)
Comment (lucasmichot) [inline src/Illuminate/Auth/AuthManager.php]: I think there are many other inline assignments in the project.
Or do you mean here ?

Comment (GrahamCampbell) [inline src/Illuminate/Auth/AuthManager.php]: I'm meaning I personally avoid them, and I think Taylor generally does too. There's no real benefit to doing it here, so I'd suggest we don't. :)

**Comment by vlakoff** [comment]
+1 for avoiding these inline assignments.

**Comment by taylorotwell** [comment]
It was working before and it's so close to release I don't want to dig around making formatting type changes in case we break something.

**Pull request closed without merging**